tests/test_gc.rs

```diff
@@ -110,7 +111,7 @@ fn create_pointers_in_drop() {
     {
         let gil = Python::acquire_gil();
         let py = gil.python();
-        let empty = PyTuple::empty(py);
+        let empty: Py<PyTuple> = FromPy::from_py(PyTuple::empty(py), py);
         ptr = empty.as_ptr();
         cnt = empty.get_refcnt() - 1;
```

Comment:
`PyTuple::empty(py)` increases the refcount of the empty tuple by 1 and `FromPy::from_py(PyTuple::empty(py), py)` by 2. So if `Py<PyTuple>` is used here for `empty` (not necessary) then `cnt` has to be corrected with
```rust
cnt = empty.get_refcnt() - 2;
```